ui/apps/platform/cypress/integration/compliance/Compliance.selectors.js

```diff
@@ -4,8 +4,8 @@ export const selectors = {
             firstGroup: '.table-group-active:first',
             firstTableGroup: '.rt-table:first',
             firstRow: 'div.rt-tr-group > .rt-tr.-odd:first',
-            firstRowName: 'div.rt-tr-group > .rt-tr.-odd:first .rt-td a',
-            secondRowName: 'div.rt-tr-group > .rt-tr.-even:first .rt-td a',
+            firstRowName: 'div.rt-tr-group > .rt-tr.-odd:first .rt-td div',
+            secondRowName: 'div.rt-tr-group > .rt-tr.-even:first .rt-td div',
```

Comment:
A little confused about this one. Were the tests failing before? Was there a change in the Compliance v1 feature recently?

Reply:
Yeah, looks like we still use the `a` selector on master too. I'll approve the PR. I'll bring it up with the UI team as well.